snapcraft/internal/meta/hooks.py

```diff
@@ -14,20 +14,32 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from collections import OrderedDict
 import re
 
-from copy import deepcopy
 from snapcraft.internal.meta.errors import HookValidationError
-from typing import Any, Dict
+from typing import Any, Dict, List, Optional
```

Comment:
This import here does not follow PEP8, there is no check for this, but we do follow it across the codebase: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Fixed!